systemtest/src/main/java/io/strimzi/systemtest/resources/kubernetes/ServiceResource.java

```diff
@@ -38,7 +38,7 @@ public boolean waitForReadiness(Service resource) {
     }
 
     public static Service createServiceResource(ExtensionContext extensionContext, Service service, String clientNamespace) {
-        LOGGER.info("Creating Service {} in namespace {}", service.getMetadata().getName(), clientNamespace);
+        LOGGER.info("Creating Service {} in Namespace {}", service.getMetadata().getName(), clientNamespace);
```

Comment:
are we going to use format `Kind: <name>` e.g. Namespace: namespace-0
or `Kind <name>` e.g. Namespace namespace-0 
in this case there is not symol `:` used, in change in  
systemtest/src/main/java/io/strimzi/systemtest/resources/kubernetes/RoleResource.java it is (this is change in file above).